calico_rkt/calico_rkt.py

```diff
@@ -201,9 +207,12 @@ def _create_profile(endpoint, profile_name, ip, client):
         })
     print(dump)
 
-def _create_rules(id_):
+def _create_rules(profile):
```

Comment:
Since these are the default rules for rkt (and they may change), lets rename this `_create_default_rules`